elastalert/alerts.py

```diff
@@ -320,12 +322,22 @@ def __init__(self, rule):
         self.jira_args = {'project': {'key': self.project},
                           'issuetype': {'name': self.issue_type}}
 
+        if self.security_level:
+            self.jira_args['security'] = {'name': self.security_level}
```

Comment:
If the field is actually called 'security' in the API, it might be better to expose it to the user as 'security' too, so that there is no ambiguity